models/access_token.js

```diff
@@ -0,0 +1,22 @@
+const Sequelize = require("sequelize");
+const sequelize = require("../database/db");
+const User = require("./users");
+
+const Access_Token = sequelize.define("Access_Tokens", {
+  user_id: {
+    type: Sequelize.INTEGER,
+
+    references: {
+      model: User,
+      key: "id",
+    },
+  },
+  access_token: {
+    type: Sequelize.STRING,
+  },
+  expiry: Sequelize.STRING,
```

Comment:
this should be type of date & time because as per the code your are storing date and time

  const expiryTime = new Date();
      const exTime = expiryTime.setHours(expiryTime.getHours() + 1);

Reply:
should i give it type of DATE or Time?

Reply:
what type of data is going to store in it?

Reply:
In this we are extracting time using date function.

Reply:
then use time type

Reply:
done sir.